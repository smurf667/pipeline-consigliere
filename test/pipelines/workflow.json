{
  "error": 0,
  "warn": 0,
  "info": 1,
  "fixes": 1
}
