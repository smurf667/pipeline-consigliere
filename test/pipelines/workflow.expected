workflow:
  auto_cancel:
    on_new_commit: interruptible
  rules:
    - if: $CI_PIPELINE_SOURCE == "merge_request_event"
